src/main/kotlin/com/exactpro/th2/codec/fixng/FixNgCodecFactory.kt

```diff
@@ -21,14 +21,13 @@ import com.exactpro.th2.codec.api.IPipelineCodec
 import com.exactpro.th2.codec.api.IPipelineCodecContext
 import com.exactpro.th2.codec.api.IPipelineCodecFactory
 import com.exactpro.th2.codec.api.IPipelineCodecSettings
-import com.exactpro.th2.codec.fixng.FixNgCodec.Companion.toMessages
 import com.google.auto.service.AutoService
 
 @AutoService(IPipelineCodecFactory::class)
 class FixNgCodecFactory : IPipelineCodecFactory {
     private lateinit var context: IPipelineCodecContext
 
-    @Deprecated("Please migrate to the protocols property") override val protocol: String = PROTOCOL
+    @Deprecated("Please migrate to the protocols property") override val protocols: Set<String> = setOf(PROTOCOL)
```

Comment:
```suggestion
    override val protocols: Set<String> = setOf(PROTOCOL)
```
Also, could you please extract setOf(PROTOCOL) to a constant